diesel_cli/src/main.rs

```diff
@@ -368,6 +371,12 @@ fn run_infer_schema(matches: &ArgMatches) -> Result<(), Box<Error>> {
         })
         .collect();
 
+    let filter_regex = matches
+        .values_of("table-name-regexpes")
+        .unwrap_or_default()
+        .map(|table_name_regex| Regex::new(table_name_regex).unwrap().into())
```

Comment:
Could we print a nicer error message here if the provided string is no valid regex?

(Hint: If that map returns a `Result` it is possible to collect into a `Result<Vec<_>, _>`)